apps/perf-test-react-components/src/scenarios/Persona.tsx

```diff
@@ -10,7 +10,7 @@ const Scenario = () => (
     presence={{ status: 'available' }}
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28895&commitId=46fb65f0754fc8979533d3d8d959c4209b71aad2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|1340| Changed|
|Avatar Converged.customSize+image.normal.chromium.png|84| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|52| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3357| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
The new screenshots are correct, resolving.